.gitmodules

```diff
@@ -54,3 +54,7 @@
 [submodule "src/examples/gyro_fft/CMSIS_5"]
 	path = src/examples/gyro_fft/CMSIS_5
 	url = https://github.com/ARM-software/CMSIS_5.git
+[submodule "src/drivers/imu/bno055/BNO055_driver"]
+	path = src/drivers/imu/bno055/BNO055_driver
+	url = https://github.com/BoschSensortec/BNO055_driver
```

Comment:
I'd rather not carry more submodule dependencies if we can avoid it. How much of this driver do you need? Could the minimum be copied in tree?

Reply:
> Can you share a log of this working? https://logs.px4.io/

Not of a real flight, since my vehicle is still not working (and the weather here is not exactly good right now). 
I could pick it up and run around a bit.. :)
But I guess that is not what you want, since it would be interesting to see if the sensor is fast enough for a control loop.

Reply:
> I'd rather not carry more submodule dependencies if we can avoid it. How much of this driver do you need? Could the minimum be copied in tree?

Probably yes. I tried to avoid reinventing the wheel, but looking back on the troubles I had with making the library usable from C++ code, it might have been faster to copy the register definitions and rewrite it from scratch. Sadly I don't have that much time until Jan/Feb, so I can't promise anything soon.

Btw, thanks for the feedback!